testcontainers/Cargo.toml

```diff
@@ -28,6 +28,7 @@ log = "0.4"
 memchr = "2.7.2"
 parse-display = "0.9.0"
 pin-project-lite = "0.2.14"
+russh = { version = "0.54.3", default-features = false, features = ["ring", "rsa"] }
```

Comment:
I think we need to keep it under a feature - looks like a good candidate.
Isolated functionality and we can enable feature by default later.

It's common request to be able to use testcontainers without `ring` or `aws-lc-rs`.

Reply:
I tried modifying it to use a feature gate, but it turned out to be much more complex than I expected. I might need some time to experiment on another branch.

Additionally, I just realized that I forgot to add an HTML in `docs/` introduction for this new feature. I will add it later. Thank you for your attention.